### PR TITLE
busybox deployment yaml to apply during reconcile

### LINKED ISSUE #7: add example deployment from yaml

### PR DESCRIPTION
**Pull request by itroyano**
Fixes #7

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -61,10 +62,10 @@ help: ## Display this help.
 ##@ Build
 
 docker-build: ## Build docker image with the manager.
-	mvn clean package -Dquarkus.container-image.build=true -Dquarkus.container-image.image=${IMG}
+	mvn clean package -Pnative -Dquarkus.container-image.build=true -Dquarkus.container-image.image=${IMG} -Dquarkus.native.container-runtime=${IMAGE_BUILDER}
```
Comment (skattoju) [inline Makefile:65]: is'nt it `-Dnative`

File: Makefile [hunk 2]
```diff
@@ -1,5 +1,6 @@
 
 VERSION ?= 0.0.1
+IMAGE_BUILDER?=podman
```
Comment (skattoju) [inline Makefile]: may be leave it as docker for now ?

File: Makefile (hunk 1 above)
Comment (itroyano) [reply Makefile:65]: doesn't work.  it's -P

File: Makefile (hunk 2 above)
Comment (itroyano) [reply Makefile]: changing.

Comment (itroyano) [reply Makefile]: done.

File: Makefile (hunk 1 above)
Comment (skattoju) [reply Makefile:65]: worked for me .. even though -P is for profile and -D is for properties 🤔

**Review by skattoju — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (itroyano) [reply Makefile:65]: When you pass -D and try setting the builder to podman, it still looks for docker.
Looks like only -P activates the container-runtime setting used later in this line.

**Comment by itroyano** [comment]
also #5

**Pull request merged**